projects/demo/src/modules/directives/header/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {TuiComponentPipe, TuiExamplePipe, TuiSetupComponent} from '@demo/utils';
+import type {TuiRawLoaderContent} from '@taiga-ui/addon-doc';
+import {TuiAddonDocModule, tuiDocExampleOptionsProvider} from '@taiga-ui/addon-doc';
+
+@Component({
+    standalone: true,
+    imports: [TuiExamplePipe, TuiComponentPipe, TuiAddonDocModule, TuiSetupComponent],
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection,
+    providers: [tuiDocExampleOptionsProvider({fullsize: true})],
+})
+export default class ExampleComponent {
+    protected readonly exampleModule: TuiRawLoaderContent = import(
```

Comment:
nit: I try to name those `import` and `template` so they are named the same across pages. We can rename the files as well. Makes it easier to copy-paste pages.

Reply:
what about

```html
[code]="'insert-module' | tuiSetup"
```

```html
[code]="'insert-template' | tuiSetup"
```

Reply:
I've renamed it for now, we can think about the pipe in a separate PR